### ISSUE TITLE
train，GPU利用率太低

### ISSUE THREAD

**Issue by haobabuhaoba** [argusswift/YOLOv4-pytorch#58]
Windows10 I57500 gpu 2070 训练两类共1000张图片，为什么训练这么慢啊？
batch_size=8
采用默认的--weight_path",
   default="weight/mobilenetv3.pth",
使用GPU训练，这是第64次迭代，共120次
:inference time: 17.95 ms
[2020-10-23 11:03:56,519]-[train.py line:288]:save weights done
[2020-10-23 11:03:56,519]-[train.py line:289]:  ===test mAP:0.914
[2020-10-23 11:03:56,519]-[train.py line:305]:  ===**cost time:1302.5833s**
GPU的利用率太低了，我每4秒监测一次Gpu,发现利用率最多到12左右，经常是2%左右

**Comment by haobabuhaoba** [comment]
"NUMBER_WORKERS": 0,

**Comment by argusswift** [comment]
> Windows10 I57500 gpu 2070 训练两类共1000张图片，为什么训练这么慢啊？
> batch_size=8
> 采用默认的--weight_path",
> default="weight/mobilenetv3.pth",
> 使用GPU训练，这是第64次迭代，共120次
> :inference time: 17.95 ms
> [2020-10-23 11:03:56,519]-[train.py line:288]:save weights done
> [2020-10-23 11:03:56,519]-[train.py line:289]: ===test mAP:0.914
> [2020-10-23 11:03:56,519]-[train.py line:305]: ===**cost time:1302.5833s**
> GPU的利用率太低了，我每4秒监测一次Gpu,发现利用率最多到12左右，经常是2%左右

是在nvidia-smi里监测的吗

**Comment by jinfagang** [comment]
@haobabuhaoba  I have forked a multi-gpu training version with DDP support. It's can 8x faster loss converge if you have 8 GPUs.

**Comment by cymdhx** [comment]
你好 我想问一下解决gpu利用低的问题了吗 我为什么是cpu训练的
